### DISCUSSION TITLE
Welcome to artillery-Genius Discussions!

### DISCUSSION THREAD

**Discussion by dgtaheno** [dgtaheno/artillery-Genius#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arqtron** [comment]
Hello David,

Thank you for this firmware update. A week ago I had problems connecting Octoprint with the previous firmware, RCA tech, due to a data corruption issue. Searching for an updated firmware I found this one and I have to say that I was pleasantly surprised. I haven’t made any adjustments yet but the result of the first test print is incredible. The first layer is done at a very slow speed but adhesion is assured, another problem I had lately. This first layer looks a bit wider too, I think.
I think it’s the best quality I’ve gotten with the Artillery Genius. The menus are well organized and I can connect Octoprint again.

**Reply by dgtaheno** [reply]
Hello Xavier,

Glad that the firmware works good for you!
If you might experience any issues with the firmware just let me know and I'll try to support you!

By the way if you experience adhesion issues I would recommend you to try 3D LACS(not sponsored), I also had some problems after cleaning the bed with alcohol and since I'm using it, not a single adhesion issue.

Happy printing!

BR
